lib/bitcrowd_ecto/repo.ex

```diff
@@ -121,8 +121,8 @@ defmodule BitcrowdEcto.Repo do
       end
 
       @impl BER
-      def count(queryable) do
-        BER.count(__MODULE__, queryable)
+      def count(queryable, opts \\ []) do
```

Comment:
This needs an additional `@callback` in the behaviour above. You could choose to also give the `/2` its own `@doc` and `@doc since: ...` then to make it show up in the docs by itself. IMO preferable. Please also mention what the `opts` arg is good for.